.solhint.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "solhint:recommended",
+  "plugins": [],
+    "rules": {
+      "compiler-version": ["off"],
+      "no-inline-assembly": ["error"],
+      "not-rely-on-time": ["off"],
+      "avoid-low-level-calls": ["off"],
+      "max-line-length": ["error", 150],
+      "func-param-name-mixedcase": "error",
+      "modifier-name-mixedcase": "error",
+      "reason-string": ["error", { "maxLength":60 }]
```

Comment:
I think a maxLength of 32 would be more useful because, if I remember correctly, reasons with 32 characters or less are stored as a single bytes32. So it'd bee nice to be notified when a reason needs to be shortened to 32 characters.

Reply:
Okay I've demoted it to a warning and switched the `maxLenth`